src/application/gtkrs.rs

```diff
@@ -395,22 +399,39 @@ fn _create_webview(
     for js in attributes.initialization_scripts {
         webview = webview.initialize_script(&js);
     }
-    if let Some(cbs) = callbacks {
-        for Callback { name, mut function } in cbs {
-            let proxy = proxy.clone();
-            webview = webview.add_callback(&name, move |_, seq, req| {
-                function(
-                    WindowProxy::new(
-                        ApplicationProxy {
-                            inner: proxy.clone(),
-                        },
-                        window_id,
-                    ),
-                    seq,
-                    req,
-                )
-            });
-        }
+
+    let begin_drag_callback = Callback {
+        name: "beginDrag".to_owned(),
```

Comment:
Should this be camel or snake case? On Tauri we decided that user-defined commands would be snake case, so it might be best to do that here as well.

Reply:
yeah I think that's better, I also want to rename it to something that is unlikely to be overridden by users. should I rename it to `wry_invoke_begin_drag`

Reply:
Also, it might be best to name it `__WRY_BEGIN_DRAG__` as to not interfere with another `beginDrag` function, which doesn't seem all that uncommon to me.

Reply:
yeah that's better

Reply:
Whoops, your comment didn't load till after I posted mine. Looks like we are thinking along the same lines though :joy:. Either name would be fine, but the general convention is to use `__` around all-caps I think.